mvc/src/main/java/nextstep/web/support/RequestMethod.java

```diff
@@ -1,5 +1,14 @@
 package nextstep.web.support;
 
+import java.util.Arrays;
+
 public enum RequestMethod {
-    GET, HEAD, POST, PUT, PATCH, DELETE, OPTIONS, TRACE
+    GET, HEAD, POST, PUT, PATCH, DELETE, OPTIONS, TRACE;
+
+    public static RequestMethod find(final String method) {
+        return Arrays.stream(values())
+                .filter(value -> value.name().equals(method))
+                .findAny()
+                .get();
+    }
```

Comment:
디폴트로 주어지는 `RequestMethod.valueOf(method)` 메서드 대신 직접 find를 구현해주셨군요. 

Optional에 대해 곧바로 get을 호출하고 있는데 커스텀한 예외 메시지를 보여주는 것은 어떠할까요?

```suggestion
    public static RequestMethod find(final String method) {
        return Arrays.stream(values())
                .filter(value -> value.name().equals(method))
                .findAny()
                .orElseThrow(() -> new IllegalArgumentException("지원되지 않는 메서드입니다."));
    }
```

Reply:
놓쳤던 부분이네요! 👍 설정했습니다. https://github.com/woowacourse/jwp-dashboard-mvc/pull/123/commits/94814789ffbeeda1223b06b9949979582bbb6627